src/script/conversation/ConversationState.ts

```diff
@@ -71,6 +71,12 @@ export class ConversationState {
     this.selfMLSConversation = ko.pureComputed(() =>
       this.conversations().find(conversation => isMLSConversation(conversation) && isSelfConversation(conversation)),
     );
+
+    //anytime mls conversation state changes, we update the sorted conversations that will recalculate visible conversations
+    useMLSConversationState.subscribe(() => {
+      this.sortedConversations.notifySubscribers();
+    });
```

Comment:
`sortedConversations` is used when calculating `visibleConversations`, so `visibleConversations` will recalculate every time mls conversations store gets updated.